File: c/ks.c
#include <stdio.h>
#include <time.h>

int main()
{
     struct tm *t;
     time_t now;
     float ks;
     time(&now);
     t=localtime(&now);
     ks=(t->tm_hour*3600 + t->tm_min*60 + t->tm_sec);
     printf("it is %.2f kiloseconds\n",ks/1000);
     return 0;
}

File: go/ks.go
package main

import (
	"fmt"
	"time"
)



func main() {
	now := time.LocalTime()
	seconds := (now.Hour * 3600) + (now.Minute * 60) + now.Second
	fmt.Printf("It is %.3f kiloseconds\n", (float64(seconds) / 1000))
}

